.github/workflows/build-test-deploy.yml

```diff
@@ -161,13 +161,20 @@ jobs:
 
       - name: Publish unit test results
         uses: EnricoMi/publish-unit-test-result-action@170bf24d20d201b842d7a52403b73ed297e6645b # v2.18.0
-        if: always()
+        if: ${{ !cancelled() }}
         with:
           check_name: unit-test-results
           files: |
             build/test-results/**/*.xml
             src/main/app/reports/*.xml
 
+      - name: Upload unit test results to CodeCov
```

Comment:
So you changed the order right? Now first upload unit test results to CodeCov and only then generate the code coverage report? I don't understand how that could work because but maybe I'm missing something.. ;-)

Reply:
I guess I'm missing something because I see it is working in your branch. ;-) nice! https://app.codecov.io/gh/infonl/dimpact-zaakafhandelcomponent/tree/chore%2FPZ-5193_add_codecov_test_analytics

Reply:
Order wasn't changed, this upload was just inserted into the flow.